src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
+  all,
+  active,
+  completed,
```

Comment:
```suggestion
export enum Status {
  All,
  Active,
  Completed,
```
Enum properties should be capitalized
https://www.typescriptlang.org/docs/handbook/enums.html